docs/machines/lanl.rst

```diff
@@ -88,6 +88,7 @@ grizzly, gnu
 
 .. code-block:: bash
 
+    module use /usr/projects/climate/SHARED_CLIMATE/modulefiles/all
```

Comment:
Isn't this what just stopped working on Grizzly after the latest DST?

Reply:
It doesn't work without it for gnu. It was Intel 19 that changed
```
gr-fe3:~$ module load gcc/5.3.0

gr-fe3:~$ module load openmpi/1.10.5 netcdf/4.4.1 parallel-netcdf/1.5.0 pio/1.7.2
Lmod has detected the following error:  The following module(s) are unknown: "pio/1.7.2" "parallel-netcdf/1.5.0" "netcdf/4.4.1"

Please check the spelling or version number. Also try "module spider ..."
It is also possible your cache file is out-of-date; it may help to try:
  $ module --ignore-cache load "pio/1.7.2" "parallel-netcdf/1.5.0" "netcdf/4.4.1"

Also make sure that all modulefiles written in TCL start with the string #%Module
```

Reply:
Okay, I see.  So se still get some modules that we built ourselves for gnu.  That makes sense.